pkg/cmd/cli/describe/chaindescriber.go

```diff
@@ -92,6 +93,15 @@ func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag, includeInputImag
 		return "", NotFoundErr(fmt.Sprintf("%q", ist.Name))
 	}
 
+	markers := buildanalysis.FindCircularBuilds(g, osgraph.DefaultNamer)
+	if len(markers) > 0 {
+		for _, marker := range markers {
+			if strings.Contains(marker.Message, ist.Name) {
+				return fmt.Sprintf("Exiting examination of %s. %s", ist.Name, marker.Message), nil
```

Comment:
thanks, that output looks good to me, i don't think the "Exiting ......" prefix is needed.
I do wonder if Cycle should be lower case.  defer to @smarterclayton